ui/components/GrafanaComponent.js

```diff
@@ -71,10 +71,10 @@ class GrafanaComponent extends Component {
       urlError: false,
 
       grafanaConfigSuccess,
-
+      grafanaBoardSearch: '',
       grafanaURL,
       grafanaAPIKey,
-      grafanaBoardSearch: '', // we probably dont need this retrieved from store
+      grafanaBoardSearch,
```

Comment:
I think the grafanaBoardSearch variable comes twice in this.state
Please rectify this issue @kushthedude

Reply:
First one was retrieved from store which is not needed & is unused-var.

Reply:
@kushthedude I guess I couldn't explain it better. Actually we don't want `this.state` (or any js object) to have the same key twice.
![image](https://user-images.githubusercontent.com/25280541/80770116-f583b600-8b6c-11ea-9e34-ffa4a8e04185.png)
(see line 74 and 77).

This is from the `GrafanaComponent.js` file from this PR: https://github.com/layer5io/meshery/blob/3470113f37609da4b56320cc5cebd59dfe8d1774/ui/components/GrafanaComponent.js

Reply:
I'd suggest you to remove the one on line 77

Reply:
This missed my eye.

Reply:
No problem :+1:   That's why we have PR reviewers :smile: